### ISSUE TITLE
Multiple custom projectiles only use one of their textures.

### ISSUE THREAD

**Issue by Rad586** [liopyu/EntityJs#11]
Report an issue or feature request

**Mod Version:**
0.2.4 (latest)

**Minecraft Version:**
1.19.2

**Minecraft Modloader:**
Fabric

**Describe the Bug/Feature Request:**
Multiple custom projectiles only use one of their textures.

**Screenshots or Logs:**
![image](https://github.com/user-attachments/assets/4e9fd4d2-3f33-4b67-8b79-26819583e547)
![image](https://github.com/user-attachments/assets/3a4764b5-2d55-46e0-93bf-81f069a53ef1)

**Additional Information:**
Environment: 
![image](https://github.com/user-attachments/assets/b7919ca3-5560-4eb3-bec7-7877e24a36de)

Texture files are put in these locations, which should work as expected: 
`kubejs\assets\kubejs\textures\item` and `kubejs\textures\entity\projectiles`
script:
```js
StartupEvents.registry('entity_type', e => {
    e.create('dynamite', 'entityjs:projectile')
        .item(item => item.canThrow(true))
        .sized(0.4, 0.4)
        .renderScale(0.4, 0.4, 0.4)

    e.create('sticky_torch', 'entityjs:projectile')
        .item(item => item.canThrow(true))
        .sized(0.5, 0.5)
        .renderScale(0.5, 0.5, 0.5)

    e.create('sticky_ender_pearl', 'entityjs:projectile')
        .item(item => item.canThrow(true))
        .sized(0.5, 0.5)
        .renderScale(0.5, 0.5, 0.5)
})
```

**Comment by Rad586** [comment]
[kubejs.zip](https://github.com/user-attachments/files/17600486/kubejs.zip)
Here's a kubejs folder for reproducing this issue.

**Comment by liopyu** [comment]
Alright, I'll take a look

**Comment by liopyu** [comment]
i believe its supposed to be textures/item folder the item textures go in which is what the projectile renderer renders from. you currently have your item textures in the textures/entity/item folder. It needs to be the same as base kubejs and where it reads item textures from

**Comment by liopyu** [comment]
that all being said it still is defaulting to a random texture so this is definitely a bug

**Comment by liopyu** [comment]
This was a bug on my end with the renderer, a fix should be out within an hour

**Comment by Rad586** [comment]
I accidentally messed up the file location in the file I upload, the textures are actually in the right location on my side.
Thanks for taking a further look into this issue!

**Comment by liopyu** [comment]
Alright this should be fixed as of [EntityJS 1.19.2-0.2.5](https://legacy.curseforge.com/minecraft/mc-mods/entityjs/files/5865621)

**Comment by Rad586** [comment]
Thanks for the quick fix! This issue is solved!
